### PR TITLE
PermissionSettings: Simplify and fix validation

### LINKED ISSUE #690: Daily limit
if I  manually set the daily limit (Limit for spending without asking per day), then it is not saved, it disappears. 
If using a slider, it is saved. 

perm

This issue is reproduced on the development branch.
Mac OS 11.0.1
Superhero 3832d26 / v.0.4.2 
 Superhero ca07c27 / v.0.4.2

### PR DESCRIPTION
**Pull request by davidyuk**
closes #690 was broken in 770acec1f40e3874a480fa33d19ab338c679707e

### CODE REVIEW

File: src/popup/router/pages/PermissionsSettings.vue [hunk 1]
```diff
@@ -85,11 +85,11 @@ export default {
     async changePermission(name, value) {
       let newValue = value;
       if (name === 'transactionSignLimit') {
-        if (Number.isNaN(newValue) || !Number.isFinite(newValue) || newValue < 0) {
```
Comment (davidyuk) [inline src/popup/router/pages/PermissionsSettings.vue:88]: @etharner did you used `Number.isFinite` to disallow typing `Infinity`? I think this is an edge case that we can ignore currently

Comment (etharner) [reply src/popup/router/pages/PermissionsSettings.vue:88]: It appears when user is typing very large number in input, but I agree we can omit this check

**Review by etharner — APPROVED** [review]

**Review by mradkov — APPROVED** [review]

**Review by Liubov-crypto — COMMENTED** [review]
Tested, now limit settings work.

Btw, When I set a spending limit, sent tokens are not displayed in the "Spent Today" section.

perm2
sent

**Comment by davidyuk** [comment]
@Liubov-crypto please create a separate issue for that bug

**Comment by Liubov-crypto** [comment]
> @Liubov-crypto please create a separate issue for that bug

moved to #704

**Pull request merged**